moodle/Tests/Sniffs/Commenting/fixtures/MissingDocblock/docblock_with_multiline_attributes.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/**
+ * This file contains multiple testcases for multi line attributes.
+ */
+
+namespace MoodleHQ\MoodleCS\moodle\Tests\Sniffs\PHPUnit;
+
+defined('MOODLE_INTERNAL') || die(); // Make this always the 1st line in all CS fixtures.
```

Comment:
Maybe, we can create an issue to get rid of these. I think that we don't need them anymore in the context of `moodle-cs` as a standalone product.

Reply:
I've created #135 about to remove those old `MOODLE_INTERNAL` lines.